system-variables.md

```diff
@@ -429,6 +429,14 @@ This variable is an alias for `last_insert_id`.
 - Default value: `ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION`
 - This variable controls a number of MySQL compatibility behaviors. See [SQL Mode](/sql-mode.md) for more information.
 
+### sql_require_primary_key <span class="version-mark">New in TBD</span>
+
+- Scope: SESSION | GLOBAL
+- Default value: "OFF"
+- This variable, when turned "ON" will prevent table creation without primary keys and dropping of primary keys in a cluster. Behaves similar to sysvar
+  [sql_require_primary_key](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_sql_require_primary_key) in MySQL 8.0
```

Comment:
I think the behavior of what "preventing" needs to be better described (returns an error):
```suggestion
- When enabled, this variable enforces that tables must have a primary key. Attempting to create or alter a table to not have a primary key will return an error.
- This feature is based on the similarly named [`sql_require_primary_key`](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_sql_require_primary_key) in MySQL 8.0.
```